pkg/server/server.go

```diff
@@ -266,6 +274,30 @@ func (s *Server) backup(backupDirectoryID string, policyID string) error {
 	return nil
 }
 
+func (s *Server) restore(recoveryPointID string, destDir string) error {
```

Comment:
Update state of restore to mqtt server like backup action.

Reply:
Fixed